lib/client-api/src/story_store.ts

```diff
@@ -343,10 +343,8 @@ export default class StoryStore {
         'Cannot add a story when not configuring, see https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#story-store-immutable-outside-of-configuration'
       );
 
-    if (storyParameters) {
-      checkGlobals(storyParameters);
-      checkStorySort(storyParameters);
-    }
```

Comment:
This is technically unrelated, but DeepScan warned me about it. `storyParameters` has a default value, so checking if it exists is useless. Later in this file, this variable is also used without existence check, so we can be sure it indeed is never null. I did a quick reference check to be sure.